DesktopEdge/Models/ZitiIdentity.cs

```diff
@@ -54,16 +65,25 @@ public ZitiIdentity()
 				Name = (string.IsNullOrEmpty(id.Name) ? id.FingerPrint : id.Name),
 				Status = id.Status,
 				IsMFAEnabled = id.MfaEnabled,
-				MFAInfo = new MFA() { 
+				MFAInfo = new MFA() {
 					IsAuthenticated = !id.MfaNeeded,
 				},
+				MinTimeout = id.MinTimeout,
+				MaxTimeout = id.MaxTimeout,
+				LastUpdatedTime = id.MfaLastUpdatedTime,
+				TimeoutMessage = ""
 			};
 
+			if (!zid.IsMFAEnabled) zid.IsTimingOut = false;
+
 			if (id.Services != null) {
 				foreach (var svc in id.Services) {
 					if (svc != null) {
 						var zsvc = new ZitiService(svc);
 						zid.Services.Add(zsvc);
+						if (zid.IsMFAEnabled) {
+							if (zsvc.TimeoutRemaining>-1 && zsvc.TimeoutRemaining<1260) zid.IsTimingOut = true;

```

Comment:
1260??? 21 minutes? this is strange

Reply:
same comment wrt all the other "1260" values

Reply:
Jeremy, Is this zsvc.TimeoutRemaining field updated in the UI? If the user set the frequency to 20 minutes, the first notification may come between 40-20 minutes, second notification may come between 20-0 minutes.

Reply:
It should be 1200 for 20 minute to show the little icon next to the Service count. 

The service notification is going away right? If we want the UI to handle all the timeout logic for this those notifications will just mess up the sate of things so they are commented out in the next commit